### PR TITLE
Test/system tests in tox

### PR DESCRIPTION
**Pull request by texasaggie97**
[X] This contribution adheres to [CONTRIBUTING.md]
### What does this Pull Request accomplish?
* Add system tests to tox
* Add helper script to install locally built wheel

### Why should this Pull Request be merged?
* Enables automated system tests

### What testing has been done?
* Travis

### CODE REVIEW

**Comment by texasaggie97** [comment]
test this please

**Comment by texasaggie97** [comment]
test this please

File: tools/install_local_wheel.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    fileGroup.add_argument(
+        "--driver",
+        action="store", dest="driver", default=None, required=True,
+        help="Source file")
+
+    verbosityGroup = parser.add_argument_group("Verbosity, Logging & Debugging")
+    verbosityGroup.add_argument(
+        "--exporting",
+        action="store_true", dest="exporting", default=False,
+        help="Are we building an official export? If false, don't actually interact with perforce."
```
Comment (marcoskirsch) [inline tools/install_local_wheel.py]: What is this about Perforce?

File: tox.ini [hunk 2]
```diff
@@ -16,6 +16,7 @@ description =
     flake8: Run static analysis
     docs: Generate documentation
     pkg: Verify the package
+    system_tests: Run system tests (requires driver to be installed)
```
Comment (marcoskirsch) [inline tox.ini]: this refers to the driver runtime exclusively, right? (not the bindings). Can you clarify?

File: tools/install_local_wheel.py (hunk 1 above)
Comment (texasaggie97) [reply tools/install_local_wheel.py]: Removed that option since it doesn't apply here

File: tox.ini (hunk 2 above)
Comment (texasaggie97) [reply tox.ini]: Added runtime

**Comment by texasaggie97** [comment]
test this please

**Pull request merged**